classes/AddressFormat.php

```diff
@@ -284,9 +298,9 @@ protected static function _setOriginalDisplayFormat(&$formattedValueList, $curre
         }
     }
 
-    /*
-    ** Cleaned the layout set by the user
-    */
+    /**
```

Comment:
Yeah, I'm trying to upgrade to Symfony standards. This also works much better with PhpStorm btw.